modules/custom/az_paragraphs/src/AZBackgroundImageCssHelper.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+namespace Drupal\az_paragraphs;
+
+use Drupal\Component\Render\FormattableMarkup;
+
+/**
+ * Class AZBackgroundImageCssHelper. Adds service to form background image CSS.
+ */
+class AZBackgroundImageCssHelper {
+
+  /**
+   * @return array
+   *   An array with all required settings.
+   */
+  public static function defaultSettings() {
+    return [
+      'css_settings' => [
```

Comment:
Just curious, are you thinking that other keys will be added to this associative array later on?  If not, maybe it could just be a one-dimensional array and the method could be renamed `defaultCssSettings()`?

Reply:
This can be a one dimensional array, and I can alter how it is added down here: https://github.com/az-digital/az_quickstart/pull/1059/files/1f2902aaf56afdf00ab7a9fc0d81facc092aa9bd#diff-4ebed9b1a9ecd773b91c632f8ef9212dc50473feba85ca4b9363a1f23c006bbfR68